### PR TITLE
docs: encryption in README and code example

### PR DESCRIPTION
**Pull request by kolyasapphire**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,11 +38,19 @@ Options are configured via environment variables.
...
+Exit messages are encrypted and decrypted following [EIP-2335](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2335.md) spec.
+
+You can check a simple example in TS in `encrypt.ts`.
+
 ## Running
 
 Either:
 
-- Use Docker image from Docker Hub: TODO
+- Use a Docker image from [Docker Hub](https://hub.docker.com/r/lidofinance/validator-ejector)
```
Comment (avsetsin) [inline README.md:53]: I would propose to fix the hash of the dockerimage in docs, like
`lidofinance/validator-ejector@sha256:8a68fac91d29cebecd198c14f73d7b50f91e3a5c3c9e99015a52ab644a6b2257`

File: encrypt.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { readFile, writeFile } from 'fs/promises'
+import { create } from '@chainsafe/bls-keystore'
+
+const PASSWORD = ''
```
Comment (avsetsin) [inline encrypt.ts:4]: Maybe put it in cli args or env that we could use this example?

Comment (avsetsin) [reply encrypt.ts:4]: And describe in readme how to run the command. I guess we will need to add in package.json in this case

File: README.md (hunk 1 above)
Comment (kolyasapphire) [reply README.md:53]: I agree, but for release versions. Right now we only have `dev` versions for testing.

File: encrypt.ts (hunk 2 above)
Comment (kolyasapphire) [reply encrypt.ts:4]: That's just a super simple example for now. No one will encrypt messages one by one, so it needs modifications anyway. But it's better than nothing. I propose to either leave it as-is or make a complete example with folder reading, pass in env, filename preservation and so on.

**Review by avsetsin — APPROVED** [review]

File: encrypt.ts (hunk 2 above)
Comment (kolyasapphire) [reply encrypt.ts]: Added a complete script, please take a look

**Review by avsetsin — APPROVED** [review]

**Pull request merged**